packages/fe-container/src/components/top-bar/TopBar.tsx

```diff
@@ -14,31 +14,39 @@
  * limitations under the License.
  */
 import React, {useContext} from 'react'
+import {Divider} from 'antd'
 
 import {BurgerIcon} from '@components/burger-icon/BurgerIcon'
+import {HelpIcon} from '@components/help-icon/HelpIcon'
 import {ConfigurationContext} from '@contexts/Configuration.context'
 import {UserMenu} from '@components/user-menu/UserMenu'
+import {UserContext} from '@contexts/User.context'
 
 import './TopBar.less'
-import {UserContext} from '@contexts/User.context'
 
 export const TopBar: React.FC = () => {
   const configuration = useContext(ConfigurationContext)
   const user = useContext(UserContext)
+  const mustShowBurgerIcon = (configuration?.plugins || []).length > 0
```

Comment:
```suggestion
  const mustShowBurgerIcon = (configuration?.plugins || []).length > 1
```

The icon should be visible if we have at least 2 plugins, not al least 1.